### PR TITLE
API Integration

### PR DESCRIPTION
**Pull request by ffasitella**
### Summary

- Adding integration with OpenLibrary
- Adding endpoint to get information of the books (/api/v1/books/search/:isbn)

### Trello card

https://trello.com/c/I1kfueCn/35-integraci%C3%B3n-api-externa

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Documentation:
   Enabled: false
 LineLength:
-  Max: 99
+  Max: 100
```
Comment (edkajimenez) [inline .rubocop.yml]: why?

Comment (ffasitella) [reply .rubocop.yml]: I forgot to remove that :sweat_smile:

File: app/services/external_api_service.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class ExternalApiService
```
[the remark this answers is not in the document]
Comment (edkajimenez) [reply app/services/external_api_service.rb]: I agree with this

File: app/controllers/api/v1/open_library_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      def show
+        book = search_book
+        return book_not_found if book.blank?
+
+        render json: book
+      end
+
+      private
+
+      def search_book
```
Comment (marcegl91) [inline app/controllers/api/v1/open_library_controller.rb]: It seems that this is responsibility of the service, not the controller. If you read the Wolox guide of projects structure you will find that we like controllers to be slim. The controller just needs to redirect the data and nothing more.

**Pull request merged**